spec.md

```diff
@@ -449,18 +449,17 @@ Procedure:
 
 The PreBlindSign algorithm allows a holder of a signature to blind messages that when signed, are unknown to the signer.
 
-The algorithm takes in a generated blinding factor that is used to un-blind the signature from the signer, and a pedersen commitment from the generators in the signers public key PK and a vector of messages.
+The algorithm returns a generated blinding factor that is used to un-blind the signature from the signer, and a pedersen commitment from the generators in the signers public key PK and a vector of messages.
 
 ```
-(s', commitment) = PreBlindSign((msg[i],...,msg[U]), h0, (h[i],...,h[U]))
+(s', commitment) = PreBlindSign((msg[1],...,msg[U]), h0, CGens)
 ```
 
 Inputs:
 
-- msg\[i\],...,msg\[U\], octet strings of the messages to be blinded.
+- msg\[1\],...,msg\[U\], octet strings of the messages to be blinded.
 - h0, octet string.
-- h\[i\],...,h\[U\], octet strings of generators for the messages to
-    be blinded.
+- CGens, vector of octet strings of generators for the messages to be blinded
```

Comment:
When we say "octet string" here are we meaning a string with some form of defined encoding like hexadecimal or instead are we assuming any parameters that are binary in nature be supplied as byte array?

Reply:
My understanding is that currently by octet string we mean just a byte array with no specific encoding, mainly used to represent points in extension fields using the format described [here ](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-pairing-friendly-curves-10) in [subsection 2.5](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-pairing-friendly-curves-10#section-2.5), or for representing nonnegative integers using the conversions described [here](https://www.rfc-editor.org/rfc/rfc8017.txt) at section 4.